### PR TITLE
Migrate listener group

### LINKED ISSUE #48: Migrate listener group
- [ ]  listener props
- [ ]  listener details
- [ ]  listener fields
- [ ]  Migrate field injection inside listeners group
  - [ ]  field name
  - [ ]  field type
  - [ ]  field value

### PR DESCRIPTION
**Pull request by barmac**
Closes #48

### CODE REVIEW

**Comment by barmac** [comment]
Good news: I have a functioning implementation of this. Tomorrow, I should be able to open this for review.

File: test/TestHelper.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import {
   fireEvent
 } from '@testing-library/preact';
 
+import 'preact/debug';
```
Comment (MaxTru) [inline test/TestHelper.js:6]: Why do we need this?

Comment (barmac) [reply test/TestHelper.js:6]: We don't, at least in this PR. However, we could evaluate using this in a separate PR.

File: src/provider/camunda-platform/utils/ExtensionElementsUtil.js [hunk 2]
```diff
@@ -20,3 +24,71 @@ export function getExtensionElementsList(businessObject, type = undefined) {
     elements.filter((value) => is(value, type)) :
     elements;
 }
+
+export function addExtensionElement(element, businessObject, extensionElement, bpmnFactory, commandStack) {
```
Comment (MaxTru) [inline src/provider/camunda-platform/utils/ExtensionElementsUtil.js:28]: Having this makes so much sense :-)

**Review by MaxTru — APPROVED** [review]
Amazing to see how the puzzle pieces fall together and we can easily re-use FieldInejctionProps and ScriptProps in this PR.

LGTM!

**Comment by barmac** [comment]
I've tweaked the label so that it looks good IMO. Ready for review.

image

**Comment by MaxTru** [comment]
> I've tweaked the label so that it looks good IMO. Ready for review.
> 
> image

As discussed in our meeting: LGTM :-) Can be merged IMO

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

**Pull request merged**